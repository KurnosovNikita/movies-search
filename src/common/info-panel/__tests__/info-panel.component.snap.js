import React from 'react';
import {shallow} from 'enzyme';
import InfoPanel from '../info-panel.component';

describe('InfoPanel component', () => {
    let props;
    beforeEach(() => {
        props = {
            moviesQuantity: jest.fn(),
            handleSortLinkClick: jest.fn(),
            activeSortByLink: jest.fn(),
        };
    });

    it('should render <InfoPanel> correctly', () => {
        const component = shallow((<InfoPanel {...props} />));

        expect(component).toMatchSnapshot();
    });
});
