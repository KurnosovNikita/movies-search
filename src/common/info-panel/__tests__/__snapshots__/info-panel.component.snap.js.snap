// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPanel component should render <InfoPanel> correctly 1`] = `
<div
  className="info-panel-container"
>
  <div
    className="movies-quantity"
  >
    <Component />
     movies found
  </div>
  <div
    className="sort-by-container"
  >
    <span
      className="sort-by-title"
    >
      Sort By
    </span>
    <Link
      className="sort-link"
      id="releaseDate"
      onClick={[MockFunction]}
      text="release date"
    />
    <Link
      className="sort-link"
      id="rating"
      onClick={[MockFunction]}
      text="rating"
    />
  </div>
</div>
`;
